app/main/helpers/direct_award_helpers.py

```diff
@@ -24,6 +24,9 @@ def get_direct_award_projects(user_id, return_type="all", sort_by_key=None):
         if sort_by_key:
             res['open_projects'].sort(key=itemgetter(sort_by_key))
             res['closed_projects'].sort(key=itemgetter(sort_by_key))
+        else:
+            res['closed_projects'].sort(key=itemgetter('lockedAt'), reverse=True)
```

Comment:
Won't this reverse=True negate the latest_first parameter? We shouldn't need this if we're letting people specify the chronological order directly through a method parameter.

Reply:
on the closed projects, we don't show the `Created` column therefore it needed to be reordered based on `lockedAt`.

Reply:
My bad, yes, makes sense.

Reply:
I think this method throws me off a bit altogether. Considering it's only used in two places, it feels like it's been abstracted too soon and the required logic could have just sat in the two places it's called. There's some dissonance around sorting closed_projects by lockedAt, I guess, which means it's not instantly apparent that these attributes are one and the same.

Also when we call this function in `save_search`, we pass a sort_by_key of name, and then re-sort the list on name again afterwards. We could take that out at the least. :)

Reply:
I could remove the additional sort from here and put it into the function itself. Other than this specific line, everything else is generic.